server/pairing/client.go

```diff
@@ -429,31 +429,40 @@ func (c *ReceiverClient) sendInstallationData() error {
 }
 
 // setupReceivingClient creates a new ReceiverClient after parsing string inputs
-func setupReceivingClient(backend *api.GethStatusBackend, cs, configJSON string) (*ReceiverClient, error) {
+func setupReceivingClient(backend *api.GethStatusBackend, cs, configJSON string) (*ReceiverClient, *ReceiverClientConfig, error) {
 	ccp := new(ConnectionParams)
 	err := ccp.FromString(cs)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	conf := NewReceiverClientConfig()
 	err = json.Unmarshal([]byte(configJSON), conf)
 	if err != nil {
-		return nil, err
-	}
-	err = validateAndVerifyNodeConfig(conf, conf.ReceiverConfig)
-	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
-	conf.ReceiverConfig.DB = backend.GetMultiaccountDB()
+	if conf.ReceiverConfig.TransferringKeystoreFiles {
```

Comment:
I believe this and following validation can be moved into `server/pairing/common.go` 🤔